Cargo.toml

```diff
@@ -20,15 +20,15 @@ failure = "0.1.5"
 name = "runqlat"
 path = "src/runqlat/main.rs"
 
+[[bin]]
+name = "tcpconnect"
+path = "src/tcpconnect/main.rs"
+
 [features]
 static = ["bcc/static"]
-bcc_v0_4_0 = ["bcc/v0_4_0"]
```

Comment:
Not sure why the changes starting from here are showing in this diff. But I think should only be adding the binary target above.